Add user-facing docs for K8s version support (#629)
submariner-io/submariner-website: src/content/getting-started/_index.en.md

* [Calico](https://www.projectcalico.org/) (see the [Calico-specific deployment instructions](../operations/deployment/calico/))
* [OVN](https://github.com/ovn-org/ovn-kubernetes)

> Submariner supports all currently-supported Kubernetes versions, which are the [three most recent
> releases](https://kubernetes.io/releases/).

Reviewer: ```suggestion
Submariner supports all currently-supported Kubernetes versions, as determined by [the Kubernetes release policy](https://kubernetes.io/releases/).
```

to make it explicit that we’re following K8s policy and to avoid having to change the text if the policy changes.
